companies/named-company.json

```
@@ -0,0 +1,11 @@
+{
+    "slug": "named-company",
+    "address": "NamedCompany\nStraße 5\n 1324 Stadt",
```

Beeop Boop I am a bot and this is my suggestion
```suggestion
    "address": Straße 5\n 1324 Stadt"
```